### DISCUSSION TITLE
[QoL Request] Open site path folder from Sites list

### DISCUSSION THREAD

**Discussion by DarrenChowles** [beyondcode/herd-community#657] [Ideas]
It would be really useful if we could open the folder either by clicking the site Path, or with a new additional icon:

![path](https://github.com/beyondcode/herd-community/assets/8046869/b7f0ccab-b3c7-4851-987f-29e9a1d5be36)

**Comment by andielliott** [comment]
Only came here to suggest the same thing, it would be really useful.

**Comment by DarrenChowles** [comment]
Or something like this that will allow us to open the sites root folder quickly from the tray menu. WAMP has this, and it's a minor but helpful feature.

![folder](https://github.com/beyondcode/herd-community/assets/8046869/bb74dc88-a590-42d3-9c4d-7e88dcdc791c)
